README.md

```diff
@@ -29,6 +83,9 @@ expect(tree).toMatchStyledComponentsSnapshot()
 
 ## toHaveStyleRule [React]
 
+Only checks for the styles directly applied to the component it receives, to assert that a style has been applied to a child
```

Comment:
If you assign a style to a component from a library ( a component that you do not own), would you use toMatchSnapshot or would you try and find it, esp. if unstyled and styled elements of the same target elementt are in the subject component.